### ISSUE TITLE
Core upload field <meta> XML should show file system date

### ISSUE THREAD

**Issue by cylkee** [symphonycms/symphony-2#2653]
Hi,

This is a suggestion.  I've got a situation where I would like to know the file system date of a file that's been uploaded with the Symphony core upload field.

Currently the XML output shows only the Symphony date in `<meta>` at the point the file was uploaded and its link created in the database.  In this situation, I'd like to create this link once, and then modify the file outside of Symphony, say FTP, and later know its modification date.

Also, the fact that `<meta>` is showing Symphony's date seems redundant to me, since I can select `system-date` to be output in the XML, e.g.

```
<system-files>
  <section id="13" handle="system">System</section>
  <entry id="69">
    <file size="15 KB" bytes="15368" path="" type="text/x-asm">
      <filename>default.css</filename>
      <meta creation="2017-03-31T11:40:55+01:00" />
    </file>
    <system-date>
      <created iso="2017-03-31T11:40:55+01:00" timestamp="1490956855" time="11:40" weekday="5" offset="+0100">2017-03-31</created>
      <modified iso="2017-03-31T11:40:55+01:00" timestamp="1490956855" time="11:40" weekday="5" offset="+0100">2017-03-31</modified>
    </system-date>
  </entry>
  <entry id="68">
    <file size="100 KB" bytes="102584" path="" type="text/html">
      <filename>default.js</filename>
      <meta creation="2017-03-31T11:40:37+01:00" />
    </file>
    <system-date>
      <created iso="2017-03-31T11:40:37+01:00" timestamp="1490956837" time="11:40" weekday="5" offset="+0100">2017-03-31</created>
      <modified iso="2017-03-31T11:40:37+01:00" timestamp="1490956837" time="11:40" weekday="5" offset="+0100">2017-03-31</modified>
    </system-date>
  </entry>
</system-files>
```

To test, I modified one of these files and the file system date/time changed accordingly, but the above Symphony date/time stayed the same.

In short, I think the `<meta>` child of `<file>` should show both filesystem **creation** and **modification** dates, with the same attribute detail as how `<system-date>` outputs, with `timestamp=` etc.

Small aside, the `type=""` doesn't seem to have found the correct type in these instances.

**Comment by michael-e** [comment]
The `meta` info is read from the database, which is correct because the file is supposed to be uploaded and handled by Symphony. It is **not** showing the entry timestamp", but a "file timestamp" which is writte to the DB when the file gets uploaded. (If you create an entry first, then upload a file, you will see this.)

Using the filesystem date would slow down the output. And it would imply the fact that an uploaded file can also be handled by processes outside of Symphony (which is undesirable, IMHO).

**Comment by cylkee** [comment]
Okay, points understood.  It's probably a niche case.  Feel free to close this issue.

**Comment by nitriques** [comment]
@cylkee If you need to change data outside of Symphony, it's a better way to let Symphony does what it should do. Try to poke at the REST API extension: This makes it easy to integrate from the outside ;)
